content/docs/coldchain/equipment.md

```diff
@@ -157,7 +151,7 @@ Clicking this button will open the `Create Asset` modal
 
 With this modal you can assign:
 
-- set the store for the asset
+- set the store for the asset (only available on central server)
```

Comment:
![image](https://github.com/msupply-foundation/msupply_docs/assets/9192912/c9e98c55-96fa-4407-91a2-20e5189567bf)

🤔

Reply:
Can we decide how to do this later... and update the places relevant.. not sure how I feel about the screenshot next to the next yet D: